### PR TITLE
Update to 2.0.0rc1

### PR DESCRIPTION
**Pull request by jtpio**
This PR should be updated to target an `rc` branch: https://github.com/conda-forge/jupyterlab_server-feedstock/issues/27

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: .ci_support/linux_.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge jupyterlab_rc
```
Comment (jasongrout) [inline .ci_support/linux_.yaml:4]: I think we shouldn't be touching this file?

**Comment by jasongrout** [comment]
It looks like for simplicity, you are targeting the `jupyterlab_rc` channel, but I think it's better to target `jupyterlab_server_rc` - that will be much clearer in the long run, I think, if we are consistent across repos with the conda-forge recommendations.

**Comment by jasongrout** [comment]
Note that one prerelease can depend on another prerelease channel: https://github.com/conda-forge/cfep/blob/master/cfep-05.md#specification

File: .ci_support/linux_.yaml (hunk 1 above)
Comment (jasongrout) [reply .ci_support/linux_.yaml:4]: Note in the [instructions](https://github.com/conda-forge/cfep/blob/master/cfep-05.md#specification):

> NOTE: A rerender needs to happen for these changes to reflect in CI files.

So we shouldn't touch these files, but instead ask for a rerender.

**Comment by jasongrout** [comment]
@conda-forge-admin, please rerender

**Comment by jasongrout** [comment]
I created an rc branch. Can you retarget this PR to the rc branch?

**Comment by jtpio** [comment]
Done.

**Comment by jtpio** [comment]
> I think it's better to target jupyterlab_server_rc - that will be much clearer in the long run

Thanks that sounds good :+1:

**Comment by jasongrout** [comment]
I think this is probably good to go, then!

**Review by jasongrout — APPROVED** [review]

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**